src/enums/ClaimConditionType.ts

```diff
@@ -0,0 +1,27 @@
+enum ClaimConditionType {
+  StartTime,
+  LimitPerWallet,
+  MerkleRoot,
+  WaitTimeBetweenClaims,
+}
+
+/**
+ * Questions:
+ *
+ * 1. Can there only be a single `startTimestamp` condition? What is a `startTimestamp` condition?
+ *
+ * 2. What is `maxMintSupply`?
+ *
+ * 3. What is `currentMintSupply`?
+ *
+ * 4. What is `quantityLimitPerTransaction`?
```

Comment:
limit how many NFTs you can mint PER TRANSACTION. 

if it's set to 1, you can only call `claim(1, proofs)` once at a time.